tests/UI/pages/FO/myAccount/addresses.js

```diff
@@ -18,7 +18,7 @@ class Addresses extends FOBasePage {
     this.addressPageTitle = 'Address';
     this.addAddressSuccessfulMessage = 'Address successfully added!';
     this.updateAddressSuccessfulMessage = 'Address successfully updated!';
-    this.deleteAddressSuccessfulMessage = 'Address successfully deleted!';
+    this.deleteAddressSuccessfulMessage = 'Address successfully deleted.';
```

Comment:
Same here, why only `Address successfully deleted!` has its exclamation point removed?

Reply:
I agree Why is the a `!` at all? First time I see this in our wording, usually we use a more neutral `.`

Reply:
Oh but wait it's in the E2E tests it's not a wording, it's a test fixture data right? Or does it need these values because the wording itself is wrong in the tested component?

Reply:
Seems like it should be fixed here as well (at least):
https://github.com/PrestaShop/PrestaShop/blob/d0f9edeb15b7c1b249d8b7fdb692987d3270f6f2/controllers/front/AddressController.php#L76-L80

Reply:
fixed here: [`97881d9` (#29187)](https://github.com/PrestaShop/PrestaShop/pull/29187/commits/97881d91f441e9dc79e8ec581bd66010b869247d)